copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>the "from this folder" link points to a specific version of the folder. Probably better to point to use a branch rather than a sha in the url.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thank you. That's coming from the existing notebooks in the guide which were used as an initial template. Maybe it would be better to create separate issue for this to update all the notebooks.</p>